src/main.ts

```diff
@@ -430,7 +437,12 @@ export default class ReadwisePlugin extends Plugin {
   ) {
     if (!this.settings.token) return;
 
-    const targetBookIds = bookIds || this.settings.booksToRefresh;
+    let targetBookIds = bookIds || this.settings.failedBooks;
+
+    // handle legacy `booksToRefresh` setting; see tsdoc on bookssToRefresh
+    if (this.settings.refreshBooks) {
+      targetBookIds = targetBookIds || this.settings.booksToRefresh;
+    }

```

Comment:
only use `booksToRefresh` if `resync deleted files` is enabled (and no `targetBookIds` already provided)

Reply:
potential concern: if failed books continue to fail, they'll always end up in targetBookIds which means that booksToRefresh would *never* get used. :thinking:

Reply:
yeah.. could you just append `this.settings.booksToRefresh` to the end of targetBookIds?

Reply:
i think the odd-man-out case would be that the `delete and reimport` command may end up importing *more* than the single book you expect. gimme a few mins to think.

Reply:
i believe i've solved this concern.